e2e/cli/e2e-npm-commons/audit.ts

```diff
@@ -12,7 +12,7 @@ export function runAudit(npm) {
       await registry.init();
     });
 
-    test.each([['verdaccio-memory', '@verdaccio/cli']])(
+    test.each([['verdaccio-memory'], ['@verdaccio/cli']])(
```

Comment:
I don't follow ... according this ... https://vitest.dev/api/#test-each is correctly implemented.

Reply:
correct syntax, but not what we want 😉